index.html

```diff
@@ -131,16 +131,22 @@
 
 ## The Hidden Cost of Bundling
 
-Bundling lowers latency and gives coherence to the whole package,
+Bundling lowers latency, lowers the protocol cost by reducing the number of
+requests and gives coherence to the whole package,
 but the hidden cost is the unused payload
 and the recurring cost of it during an update.
 There is also the inherent risk of “write amplification”,
 where even a single byte change in a web application requires
-downloading the entire application from scratch.
+downloading the entire application from scratch, which is the extreme case of the partial update issue.
```

Comment:
As a reader, I'm not sure what this means.

Reply:
```suggestion
downloading the entire application from scratch.
```